helm-fdr/helm/values-dev.yaml

```diff
@@ -29,7 +33,7 @@ fdrnodo:
     create: true
   resources:
     requests:
-      memory: "640m"
+      memory: "640M"
```

Comment:
typo ?

Reply:
with `m` it goes in error -> https://kubernetes.io/docs/concepts/configuration/manage-resources-containers/#meaning-of-memory